web/src/App.tsx

```diff
@@ -11,12 +11,14 @@ import { BrowserRouter as Router, useRoutes } from "react-router-dom";
 
 import "./App.css";
 import "antd/dist/antd.css";
+import MembersPage from "./components/pages/Members";
 
 function AppRoutes() {
   const routes = useRoutes([
     { path: "/", element: <RootPage /> },
     { path: "/dashboard", element: <DashboardPage /> },
     { path: "/dashboard/:workspaceId", element: <DashboardPage /> },
+    { path: "/members/:workspaceId", element: <MembersPage /> },
```

Comment:
Up for debate, but logically speaking wouldn't it be better if this was `/workspaces/:workspaceId/members`?
For me it makes more sense because `/members` is probably never going to make sense for us. If we were going to have `/members/id/projects` or something, okay, but we are workspace based and the members are related to workspaces.